test/Microsoft.ML.Tests/Transformers/KeyToVectorEstimatorTests.cs

```diff
@@ -58,10 +58,10 @@ public void KeyToVectorWorkout()
                     new ValueToKeyMappingEstimator.ColumnInfo("TermC", "C", textKeyValues:true)
                 }).Fit(dataView).Transform(dataView);
 
-            var pipe = new KeyToVectorMappingEstimator(Env, new KeyToVectorMappingTransformer.ColumnInfo("CatA", "TermA", false),
-                new KeyToVectorMappingTransformer.ColumnInfo("CatB", "TermB", true),
-                new KeyToVectorMappingTransformer.ColumnInfo("CatC", "TermC", true),
-                new KeyToVectorMappingTransformer.ColumnInfo("CatCNonBag", "TermC", false));
+            var pipe = new KeyToVectorMappingEstimator(Env, new KeyToVectorMappingEstimator.ColumnInfo("CatA", "TermA", false),
```

Comment:
>KeyToVectorMappingEstimator [](start = 27, length = 27)

ML.Transforms.Conversion....?

Reply:
Same here!

---
In reply to: [253665034](https://github.com/dotnet/machinelearning/pull/2364#discussion_r253665034) [](ancestors = 253665034)